src/importexport/guitarpro/internal/importgtp.cpp

```diff
@@ -2886,19 +2887,19 @@ static void addMetaInfo(MasterScore* score, GuitarPro* gp)
         }
         if (!gp->subtitle.isEmpty() || !gp->artist.isEmpty() || !gp->album.isEmpty()) {
             Text* s = Factory::createText(m, TextStyleType::SUBTITLE);
-            QString str;
+            String str;
             if (!gp->subtitle.isEmpty()) {
                 str.append(gp->subtitle);
             }
             if (!gp->artist.isEmpty()) {
                 if (!str.isEmpty()) {
-                    str.append("\n");
+                    str.append(String("\n"));
                 }
                 str.append(gp->artist);
             }
             if (!gp->album.isEmpty()) {
                 if (!str.isEmpty()) {
-                    str.append("\n");
+                    str.append(String("\n"));
```

Comment:
needs just u and maybe needs add method append(char16_t ch) to String
(actualy, here conversion const char* -> QString -> String)